acceptancetests/assess_log_rotation.py

```diff
@@ -148,7 +148,7 @@ def check_expected_backup(key, logprefix, action_output):
         raise LogRotateError(
             "Missing backup log '{}' after rotation.".format(key))
 
-    backup_pattern = "/var/log/juju/%s-(.+?)\.log" % logprefix
+    backup_pattern = "/var/log/juju/%s-(.+?)\.log.gz" % logprefix
```

Comment:
this feels like it should be \.log(.gz)?

something that says we might see it as .log or .log.gz